frontend/crd/src/app/admin/milestone-edit/milestone-edit.component.less

```diff
@@ -0,0 +1,29 @@
+@import "../../../common";
+
+mat-card-title {
+  /* CSS styles go here */
+  margin: 0px; // for example to remove the margin
+  font-weight: bold;
+}
+
+.flex-vertical-align {
+  display: flex;
+  align-items: center;
+}
+
+.mdc-button {
+  background-color: @oswego-green;
+  color:white;
+}
+
+.mat-mdc-button:not(:disabled) {
+  color:white;
```

Comment:
For future reference, most of these colors are set with CSS variables, so you can override the variable for slightly simpler selectors and better practice.

Reply:
good to know thanks! i'm still just avoiding styling until we get the official design so this will probably be redone later